src/jsx/import/navigation/menu-items/results.jsx

```diff
@@ -11,27 +11,27 @@
  *
  */
 
-import React from 'react';
+import React, { Component } from 'react';
 import { NavLink } from 'react-router-dom';
 import ajaxCaller from '../../general/ajax-caller.js';
 
-var ReviewResultsLink = React.createClass({
-  // initial 'state properties'
-    getInitialState: function() {
+class ReviewResultsLink extends Component {
+    // initial 'state properties'
+    getInitialState() {
         return {
             ajax_done_result: null,
             ajax_done_error: null,
             ajax_fail_error: null,
-            ajax_fail_status: null
+            ajax_fail_status: null,
```

Comment:
I always forget, is having the last item, followed by a comma ok? From what I remember, for json, this is generally ok, and puppet.

Reply:
Sure, it's ok in Javascript.
I think it's not ok for the JSON, though.